pkgs/development/libraries/xeus/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, doctest
+, nlohmann_json
+, libuuid
+, xtl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xeus";
+  version = "3.0.5";
+
+  src = fetchFromGitHub {
+    owner = "jupyter-xeus" ;
+    repo = pname ;
+    rev = version ;
+    sha256 = "sha256-LeU4PJ1UKfGqkRKq0/Mn9gjwNmXCy0/2SbjWJrjlOyU=" ;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    doctest
+  ];
+
+  buildInputs = [
+    nlohmann_json
+    libuuid
+    xtl
+  ];
+
+  cmakeFlags = [
+    "-DXEUS_BUILD_TESTS=ON"
+  ];
+  doCheck = true;
```

Leave a empty line here as a logical separation between build and check.
```suggestion
  ];

  doCheck = true;
```